### PR TITLE
read_liberty extensions

### PR DESCRIPTION
**Pull request by povik**
_What are the reasons/motivation for this change?_

This addresses a few things:

 * `read_liberty -wb` can only read some cells (not those with busses), but combining it with a separate `read_liberty -lib` call as fallback doesn't always work well because of how the command handles re-definitions, see commit 2f2726d with a fix
 * related to above, new `-ignore_busses` option
 * `area` and `capacitance` attributes for e.g. user inspection or to find the smallest gate implementing a particular function

Also I checked in some real-world Liberty data to use in tests, and have written tests using those (incl. for the existing `-unit_delay` option)

### CODE REVIEW

**Comment by povik** [comment]
I want to get rid of the last commit 5e2b37ffec6 checking in unfiltered data but that is blocked by #4681

**Review by widlarizer — COMMENTED** [review]
So far the PR seems sound and tested. Beware that the foundry files raw and filtered add up to over 2MB. Source exports of Yosys add up to 23M, so it's kind of a significant size change. I suggest gzipping the large files

**Comment by povik** [comment]
>  Beware that the foundry files raw and filtered add up to over 2MB

The raw files are supposed to be dropped from the PR once we fix #4681

File: frontends/liberty/liberty.cc [hunk 1]
```diff
@@ -462,6 +462,9 @@ struct LibertyFrontend : public Frontend {
 		log("    -ignore_miss_data_latch\n");
 		log("        ignore latches with missing data and/or enable pins\n");
 		log("\n");
+		log("    -ignore_busses\n");
```
[the remark this answers is not in the document]
Comment (povik) [reply frontends/liberty/liberty.cc:465]: Let me fix that then

**Comment by widlarizer** [comment]
- [ ] Check before merge that there are no commits with big files even if removed in subsequent commit to save `.git` directory size growth (I think that's how that works?)

**Review by widlarizer — APPROVED** [review]

**Pull request merged**